invokeai/frontend/dist/locales/common/ua.json

```diff
@@ -0,0 +1,53 @@
+{
+  "hotkeysLabel": "Гарячi клавіші",
+  "themeLabel": "Тема",
+  "languagePickerLabel": "Мова",
+  "reportBugLabel": "Повідомити про помилку",
+  "githubLabel": "Github",
+  "discordLabel": "Discord",
+  "settingsLabel": "Налаштування",
+  "darkTheme": "Темна",
+  "lightTheme": "Світла",
+  "greenTheme": "Зелена",
+  "langEnglish": "Англійська",
+  "langRussian": "Російська",
+  "langItalian": "Iталійська",
+  "langPortuguese": "Португальська",
+  "langFrench": "Французська",
```

Comment:
```suggestion
  "langFrench": "Французька",
```

Reply:
@netsvetaev Please go through the requested changes and let me know.